src/pointwise_loglikelihoods.jl

```diff
@@ -0,0 +1,102 @@
+"""
+    pointwise_loglikelihoods
+
+Compute pointwise conditional log-likelihoods for use in ELPD-based model comparison.
+
```

Comment:
Should we say this also for LOO validation in general, as LOO-PIT is not about model comparison?